engines/pyfunc-ensembler-service/tests/conftest.py

```diff
@@ -6,6 +6,22 @@
 
 class TestEnsembler(PyFunc):
 
+    def initialize(self, artifacts: dict):
```

Comment:
Didn't notice this previously, but perhaps we should standardize the typings? I see different options - `dict`, `Dict`, `Dict[with types]`.

Reply:
Ideally yes. But there are also other inconsistencies like double vs single quote usage. We can put in a linter at some point to catch these everywhere. For now, I just changed this example of the test ensemblers to `Dict` to match the`ensemble` method.